scripts/generate_config_files.py

```
@@ -0,0 +1,149 @@
+    "per_page": 100,  # Maximum number of repositories per page
+    "type": "public"  # Only public repositories
+}
+headers = {
+    "Accept": "application/vnd.github.v3+json"
+}
+
+matching_repos = []
+
+while url:
+    response = requests.get(url, headers=headers, params=params)
+    if response.status_code != 200:
```

Ruff (0.8.2)

Magic value used in comparison, consider replacing `200` with a constant variable

(PLR2004)